TUTORIAL.md

```diff
@@ -697,8 +720,3 @@ Flow(
 
 * [DataFlows Processors Reference](https://github.com/datahq/dataflows/blob/master/PROCESSORS.md)
 * [Datapackage Pipelines Tutorial](https://github.com/frictionlessdata/datapackage-pipelines/blob/master/TUTORIAL.ipynb) - Use the flows as building blocks for more complex pipelines processing systems.
-
```

Comment:
This is also an empty block. Just like in TUTORIAL.ipynb, I removed it, but feel free to let me know if this pull request needs any updating!